src/Resources/skeleton/security/formLogin/login_form.tpl.php

```diff
@@ -0,0 +1,44 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Log in!{% endblock %}
+
+{% block body %}
+<form method="post">
```

Comment:
Not sure if there is a standard, but I like to indent inside of my blocks. So that would mean indenting everything 4 spaces

Reply:
done